packages/stores/src/mockStore.ts

```diff
@@ -0,0 +1,202 @@
+import merge from 'lodash.merge';
+import type { TRootStore } from '../types';
+
+const mock = (): TRootStore => {
+    return {
```

Comment:
```suggestion
const mock: TRootStore = {
```
@hamid-deriv Doesn’t need to be a function.

Reply:
@farzin-deriv It needs to be a function, because when we use it in the `beforeEach`, if it's not a function it doesn't reset in render. I've tested both ways.